content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
+Suppose you’re submitting a GAUGE metric , `temperature`, from a single host running the Datadog Agent. This metric emits the following values in a 10s time interval: `[71,71,71,71,71,71,71.5]`.
 
-For example: if you send `X` values for a `HISTOGRAM` metric `<METRIC_NAME>` during an Agent flush interval, the following timeseries are produced by the Agent by default:
+The Agent submits the last reported number, in this case `71.5`, as the GAUGE metric’s value.
 
-| Aggregation                  | Description                                                                                                                                               | Datadog Metric Type |
-|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
-| `<METRIC_NAME>.avg`          | Gives you the average of those `X` values during the flush interval.                                                                                      | GAUGE               |
-| `<METRIC_NAME>.count`        | Gives you the number of points sampled during the interval, i.e. `X`. The Agent then sends it as a `RATE` so it would show in app the value `X/interval`. | RATE                |
-| `<METRIC_NAME>.median`       | Gives you the median of those `X` values in the flush interval.                                                                                           | GAUGE               |
-| `<METRIC_NAME>.95percentile` | Gives you the 95th percentile of those `X` values in the flush interval.                                                                                  | GAUGE               |
-| `<METRIC_NAME>.max`          | Gives you the maximum value of those `X` values sent during the flush interval.                                                                           | GAUGE               |
+{{% /tab %}}
+{{% tab "HISTOGRAM" %}}
 
-For instance, say that the `request.response_time.histogram` metric is reported to Datadog through an Agent with the `HISTOGRAM` type for `server:web_1` with the values [1,1,1,2,2,2,3,3] during a flush interval. The following metrics would have then be submitted to Datadog over this flush interval:
+For example, suppose you’re submitting a HISTOGRAM metric, `request.response_time.histogram`, from one web server which reports the values `[1,1,1,2,2,2,3,3]` in a defined time interval. By default, the Agent submits the following metrics to Datadog which represent the statistical distribution of these values in this time interval:
```

Comment:
Suppose you’re submitting a HISTOGRAM metric, `request.response_time.histogram`, from one web server which reports the values `[1,1,1,2,2,2,3,3]` in a flush time interval. By default, the Agent submits the following metrics to Datadog which represent the statistical distribution of these values in this time interval: